src/test_utils.rs

```diff
@@ -48,6 +63,19 @@ impl TestContext {
         t
     }
 
+    /// Create a new configured [TestContext].
+    ///
+    /// This is a shortcut which configures bob@example.net with a fixed key.
+    pub async fn new_bob() -> Self {
+        let t = Self::new().await;
+        let keypair = bob_keypair();
+        t.configure_addr(&keypair.addr.to_string()).await;
```

Comment:
This part could be moved to `configure_bob` for consistency.

Reply:
I think `.configure_alice()` is mostly historical and doesn't add any value where it is used.  They could all use `::new_alice()` instead.

I could change that in this PR if you like?